index.html

```diff
@@ -614,8 +614,7 @@ <h2>
           <p>
             Diversity for the purposes of diversity and inclusion, is any
             dimension that can be used to differentiate groups and people from
-            one another. With a focus on age, gender, ethnicity, religion,
-            disability, sexual orientation, education, and national origin.
+            one another. Diversity can include but is not limited to:  socio-economic status, sexual orientation, religion, race, physical appearance, neurotype, nationality, mental health, language, indigeneity, immigration status, gender, gender identity and gender expression, ethnicity, disability (both visible and invisible), caste, body, or age. 
```

Comment:
"body" feels ambiguous here - does it mean some organisation with whom a person may be associated, or something else?

Reply:
"body" is, with the rest of the addition, copied from another part of the CEPC. I think it is editorial to copy it as part of this PR; discussion of substance would be a separate issue.